aruna/api/storage/services/v1/info_service.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
```

Comment:
should be resource_info_service instead of info_service to separate resource logic from deployment logic

Reply:
Added by https://github.com/ArunaStorage/ArunaAPI/pull/44/commits/69f90e55728622fb82ccd1279f8a93c90e350c35